### ISSUE TITLE
What's the difference btw this and the Scene/WebScene components in react-arcgis?

### ISSUE THREAD

**Issue by tomwayson** [Esri/react-sceneview#53]
https://github.com/Esri/react-arcgis#basic-usage

**Comment by sandromartis** [comment]
Good question.

Well, it looks like `react-sceneview` supports more components, especially for tools like:

Sketch Tool: https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Sketch-SketchViewModel.html
Measurement Tool: https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-AreaMeasurement3D.html
etc.

Otherwise, it looks quite similar. What's the vision for `react-arcgis`? Do you plan to add more functionality?

And how do you handle compatibility with the ArcGIS API version in `react-arcgis`?
We often have the issue that we need to change a lot of code in `react-sceneview` when working with a newer version of the JS API. (Due to breaking changes in the API, often the measurement and sketch tools)
This then breaks it for users that still rely on an older version. (Since there is no npm installation for the JS API)

How do you handle that? You just make it work for the default version in `esri-loader`?

**Comment by tomwayson** [comment]
> What's the vision for react-arcgis? Do you plan to add more functionality?

no. I see react-arcgis as way to get React devs started using esri-loader.

> And how do you handle compatibility with the ArcGIS API version in react-arcgis?

by not adding functionality 😄 

I think it would be a good idea for react-arcgis to point users who are looking for a more full-featured component for working with scene views over here.
